Dockerfile.mysql

```diff
@@ -0,0 +1,11 @@
+FROM mysql:latest
+
+ENV MYSQL_ROOT_PASSWORD=root
+ENV MYSQL_DATABASE=app_db
+ENV MYSQL_USER=app_user
+ENV MYSQL_PASSWORD=1234
+
+VOLUME /var/lib/mysql
+
+EXPOSE 3306
+ 
```

Comment:
For correct displaying of EoF last line should be blank 
![image](https://github.com/user-attachments/assets/28ba57b8-fa1f-4ea0-99b7-194dbfb63d52)